src/main/webapp/js/upload.js

```diff
@@ -136,3 +138,24 @@ function checkFileSize() {
 
   return true;
 }
+
+/**
+ * Sets the value and max value of the transaction date input field to the
+ * current date.
+ */
+function loadDateInput() {
+  const dateInput = document.getElementById('date-input');
+  dateInput.value = dateInput.max = formatDate(new Date());
+}
+
+/**
+ * Converts a date to 'YYYY-MM-DD' format.
```

Comment:
Why this format specifically? Just curious

Reply:
This format is required to update the value of the date. The actual UI uses mm/dd/yyyy.

https://www.w3schools.com/Jsref/prop_date_value.asp